catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/SearchController.java

```diff
@@ -120,17 +120,15 @@ public void configureChannel(ConfigurableServerChannel serverChannel) {
     }
 
     /**
-     * Execute all of the queries contained within the SearchRequest
+     * Execute all of the queries contained within the {@link SearchRequest}.
      *
-     * @param request
-     *            - SearchRequest containing a query for 1 or more sources
-     * @param session
-     *            - Cometd ServerSession
+     * @param request {@link SearchRequest} containing a query for 1 or more sources
+     * @param session Cometd {@link ServerSession}
      */
     public void executeQuery(final SearchRequest request, final ServerSession session,
             final Subject subject) {
 
-        final Search search = new Search(request, actionRegistry);
+        final Search search = createSearch(request);
```

Comment:
It does not look like it would be that big of an impact to the tests if this was left the way it was but I might be missing something.